<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_measuregroup</class>
 <widget class="QWidget" name="toolbar_measuregroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QToolButton" name="pointPipeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click on a point then on a pipe.</string>
     </property>
     <property name="text">
      <string>Point - Pipe</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/cylinder_point_measurement.png</normaloff>:/icons/100x100/cylinder_point_measurement.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="threePointsPlaneCheckBox">
     <property name="text">
      <string>3 Points Plane</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="pipePlaneButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click on a pipe then a plane</string>
     </property>
     <property name="text">
      <string>Pipe - Plane</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/cylinder_plane_measurement.png</normaloff>:/icons/100x100/cylinder_plane_measurement.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="pointPlaneButton">
     <property name="toolTip">
      <string>Click on a point and then on a plane.</string>
     </property>
     <property name="text">
      <string>Point - Plane</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/point_plane_measurement.png</normaloff>:/icons/100x100/point_plane_measurement.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="pointMeshButton">
     <property name="toolTip">
      <string>Shortest distance between a mesh and a point</string>
     </property>
     <property name="text">
      <string>Point - Mesh</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/measure_points.png</normaloff>:/icons/100x100/measure_points.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="pipePipeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click on 2 pipes</string>
     </property>
     <property name="text">
      <string>Pipe - Pipe</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/cylinder_cylinder_measurement.png</normaloff>:/icons/100x100/cylinder_cylinder_measurement.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pointPlaneButton</tabstop>
  <tabstop>threePointsPlaneCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
